### ISSUE TITLE
I Got Free Chaquopy License from Mr. Malcolm from chaquopy yesterday but I still get Invalid License Error

### ISSUE THREAD

**Issue by BingolSTEM** [chaquo/chaquopy#591]
#### Relevant parts of your build.gradle file

com.chaquo.python:gradle:10.0.1

#### Describe your issue

My Android Studio Version is Arctic Fox 2020.3.1 Patch 4

I Got Free Chaquopy License from Mr. Malcolm from chaquopy yesterday but I still get Invalid License Error

I put the free license key I got into gradle's "local.properties" file as in the instructions as chaquopy.license="..............." I added.

The error I get when compiling the project is:

Execution failed for task ':app:generateDebugPythonLicenseAssets'.
> Chaquopy license verification failed

**Comment by BingolSTEM** [comment]
Oh I can't believe I solved the problem, this is great. Opencv used to be at the bottom of my app's App level build.gradle file and I guess it's based on the last post. It was fixed when I put the opencv definition on top...
this is the old one: 
applicationId "com.android.andopencvyuztanima"
applicationId "opencv.org"
this is the new one. doing so fixed it.

applicationId "opencv.org"
applicationId "com.android.andopencvyuztanima"
